component/newstorage/couchdb/store_internal_test.go

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
The name of the file should be `store_test.go`

Reply:
There's already a `store_test.go` in this PR. So this PR has two test files in `newstorage/couchdb`: `store_test.go` and `store_internal_test.go`.

`store_test.go` is a separate package and it's used to run tests against exported stuff only. My goal was to increase code coverage as much as possible using only exported stuff. However, I couldn't reach 85% coverage this way. This is where `store_internal_test.go` comes in - it's in the same package as `store.go` and allows me to test error cases that require me to mock internal components.